newIDE/app/src/EventsSheet/EventsFunctionExtractor/EventsFunctionExtractorDialog.js

```diff
@@ -65,6 +67,12 @@ export default class EventsFunctionExtractorDialog extends React.Component<
       serializedEvents,
     } = this.props;
 
+    // This is only used to check parameter for name conflict.
+    // TODO Update it according to the chosen extension.
+    this._projectScopedContainersAccessor = new ProjectScopedContainersAccessor(
+      { project }
+    );
```

Comment:
This is kind of regression as it will forbid to use the same name as global variables and global objects for a parameter.